sonarapp/Program.cs

```diff
@@ -16,16 +16,19 @@ static void Main(string[] args)
 
             Boolean whatif = (new verify()).verifywhat("whatif");
 
-            verify v= new verify();
+            Boolean whatiff = (new verify()).verifywhat("whatif");
+
+            Boolean whatifff = (new verify()).verifywhat("another whatif");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused 'whatifff' local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=csharpsquid%3AS1481)